Composer/packages/client/src/CreationFlow/DefineConversation/index.js

```diff
@@ -59,6 +75,22 @@ export function DefineConversation(props) {
     });
   };
 
+  function getDefaultName() {
+    let i = 0;
+    const bot = templateId || botName;
+    let defaultName = `${bot}-${i}`;
+
+    while (
+      bots.findIndex(bot => {
+        return bot.name === defaultName;
+      }) > -1
```

Comment:
The better way for determining whether `index === -1` or not, is using `~index`. It's called bitwase-negate. Since number in JavaScript is "two's complement", negating -1 will be 0.

```js
~(-1) === 0 //falsy
~(0) is truthy
~(1) is truthy, etc
```